index.bs

```diff
@@ -197,6 +197,8 @@ interface BarcodeDetector {
 <dl class="domintro">
   <dt><dfn constructor for="BarcodeDetector"><code>BarcodeDetector(optional BarcodeDetectorOptions |barcodeDetectorOptions|)</code></dfn></dt>
   <dd>Constructs a new {{BarcodeDetector}} with the optional |barcodeDetectorOptions|, if present.
+    * If |barcodeDetectorOptions| is passed, and its |formats| are empty, then throw a new {{TypeError}}, and abort any further steps.
+    * If |barcodeDetectorOptions| is passed, and its |formats| contain {{unknown}}, then throw a new {{TypeError}}, and abort any further steps.
```

Comment:
Since this isn't an asynchronous algorithm you don't need to say "and abort any further steps." It is implied by throwing an error.

Reply:
Done, thanks.